testing/test_conftest.py

```diff
@@ -460,7 +463,7 @@ def test_no_conftest(fxtr):
             )
         )
         print("created directory structure:")
-        tmppath = Path(str(testdir.tmpdir))
+        tmppath = Path(str(pytester.tmpdir))
```

Comment:
`testdir.tmpdir` becomes `pytester.path`; and since it's already a `Path` the conversion can be dropped.

```suggestion
        tmppath = pytester.path
```